srcpkgs/nextcloud-client/template

```diff
@@ -1,13 +1,13 @@
 # Template file for 'nextcloud-client'
 pkgname=nextcloud-client
-version=3.3.5
+version=3.4.2
 revision=1
 wrksrc="desktop-${version}"
 build_style=cmake
 configure_args="-Wno-dev"
-hostmakedepends="pkg-config inkscape"
-makedepends="qt5-tools-devel qt5-declarative-devel qt5-webchannel-devel
- qt5-location-devel qtkeychain-qt5-devel sqlite-devel libcloudproviders-devel
+hostmakedepends="pkg-config inkscape "
+makedepends="qt5-tools-devel qt5-declarative-devel qt5-webchannel-devel openssl-devel
+ qt5-location-devel qtkeychain-qt5-devel sqlite-devel sqlite libcloudproviders-devel qt5-qmake qt5-host-tools qt5-tools
```

Comment:
Please break these into new lines to keep under 80 columns.

I think the qt5 host stuff should usually go in `hostmakedepends`, too.